src/ImageRunModal.jsx

```diff
@@ -237,9 +237,10 @@ export class ImageRunModal extends React.Component {
             memoryUnit: 'MiB',
```

Comment:
Should I rename the whole dialog/file from `run` to `create`?

Reply:
You might, not necessary though

Reply:
I'll leave it as todo for later, as I feel that the PR is already quite big.